packages/terra-demographics-banner/tests/jest/DemographicsBanner.test.jsx

```diff
@@ -29,7 +28,7 @@ it('renders the banner that contains additional information', () => {
       dateOfBirth="May 9, 1993"
       gender="Male"
       identifiers={{ MRN: 12343, REA: '3JSDA' }}
-      photo={<img alt="My Cat" src="http://lorempixel.com/100/100/animals/7/" />}
+      photo={<img alt="My Cat" src="" />}
```

Comment:
Should this include the img src?
e.g. `<img alt="My Cat" src={demographicsCat} />`